tests/test_mongita.py

```diff
@@ -635,6 +635,71 @@ def test_update_many(client_class):
            set(d['weight'] + 1 if isinstance(d.get('weight'), Number) and d.get('weight') < 4 else d.get('weight') for d in TEST_DOCS)
 
 
+@pytest.mark.parametrize("client_class", CLIENTS)
+def test_push(client_class):
+    client, coll, imr = setup_many(client_class)
+    assert len(coll.find_one({"name": "Meercat"})["continents"]) == 1
+    coll.update_one({"name": "Meercat"}, {"$push": {"continents": "foobar"}})
+    new_meercat_continents = coll.find_one({"name": "Meercat"})["continents"]
+    assert len(new_meercat_continents) == 2
+    assert set(new_meercat_continents) == {"foobar", "AF"}
```

Comment:
Only thing I would say about this is that by using testing a `set` we could be potentially be making changes to the array order and not realize it. Which I believe standard Mongo preserves array order.

Reply:
You make a good point. Most Mongita unit tests make use of sets but regular MongoDB does preserve order and using sets might hide ordering bugs.